multipaz-doctypes/src/commonMain/kotlin/org/multipaz/documenttype/knowntypes/AgeVerification.kt

```diff
@@ -0,0 +1,133 @@
+package org.multipaz.documenttype.knowntypes
+
+import org.multipaz.cbor.toDataItem
+import org.multipaz.documenttype.DocumentAttributeType
+import org.multipaz.documenttype.DocumentType
+import org.multipaz.documenttype.Icon
+import org.multipaz.documenttype.knowntypes.DrivingLicense.MDL_NAMESPACE
+import org.multipaz.util.fromBase64Url
+
+/**
+ * Object containing the metadata of the Age Verification document type.
+ * See https://ageverification.dev/ for more details about this document type.
+ */
+object AgeVerification {
+    const val AV_DOCTYPE = "eu.europa.ec.eudi.pseudonym.age_over_18.1"
```

Comment:
According to https://github.com/eu-digital-identity-wallet/av-doc-technical-specification/blob/main/docs/annexes/annex-A/annex-A-av-profile.md the doctype is `eu.europa.ec.av.1`